src/components/RefinementList/RefinementListItem.js

```diff
@@ -34,11 +38,12 @@ class RefinementListItem extends React.Component {
 }
 
 RefinementListItem.propTypes = {
-  facetValue: React.PropTypes.oneOfType([
+  facetValueName: React.PropTypes.oneOfType([
```

Comment:
Yes indeed, I guess it was to avoid mixing facetValue the object (with the name property) and facetValue here which is in fact, facetValue.name.

Yes it can contain either a string or number but it is always sent as "name" to the templates. Wonder how we can make this better?